src/app/ApplicationTemplate.Shared/Business/Impl/AuthenticationService.cs

```diff
@@ -1,28 +1,26 @@
 ﻿using System;
-using System.Net.Http;
 using System.Reactive;
 using System.Reactive.Linq;
-using System.Reactive.Subjects;
-using System.Reactive.Threading.Tasks;
 using System.Threading;
 using System.Threading.Tasks;
 using ApplicationTemplate.Client;
-using MallardMessageHandlers;
 
 namespace ApplicationTemplate.Business
 {
-	public partial class AuthenticationService : IAuthenticationService, IAuthenticationTokenProvider<AuthenticationData>
```

Comment:
You can keep this recipe (`AuthenticationService` implementing `IAuthenticationTokenProvider<AuthenticationData>`) by using composition with `ConcurrentAuthenticationTokenProvider<AuthenticationData>`.